### PR TITLE
add code injection signature

### PR DESCRIPTION
**Pull request by itaysk**
cc: @roikol

### CODE REVIEW

File: tracee-rules/signatures/golang/code_injection_test.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+						{
+							Name:  "argv",
+							Value: []string{"ls"},
+						},
+					},
+				},
+			},
+			expect: true,
+		},
+	}
```
Comment (yanivagman) [inline tracee-rules/signatures/golang/code_injection_test.go:61]: Not a must, but maybe you will also want to add a  test with expect:false

File: tracee-rules/signatures/golang/helpers.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+)
+
+func GetArgumentByName(event types.TraceeEvent, argName string) (types.TraceeEventArgument, error) {
```
Comment (yanivagman) [inline tracee-rules/signatures/golang/helpers.go:10]: As we expect more input sources to be added in the future, I think that the function name should include Tracee in it

File: tracee-rules/signatures/golang/code_injection.go [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+		},
+	}, nil
+}
+
+func (sig *codeInjection) GetSelectedEvents() ([]types.SignatureEventSelector, error) {
+	return []types.SignatureEventSelector{
+		{Source: "tracee", Name: "ptrace"},
+		{Source: "tracee", Name: "open"},
+		{Source: "tracee", Name: "openat"},
+		{Source: "tracee", Name: "execve"},
```
Comment (yanivagman) [inline tracee-rules/signatures/golang/code_injection.go:44]: execveat?

Comment (itaysk) [reply tracee-rules/signatures/golang/code_injection.go:44]: I said the same thing to Roi when we wrote the code. I don't know why it was missing from the python signature, but we decided to just port the python sig as is, and @roikol should improve it if he think so

**Review by yanivagman — APPROVED** [review]

**Comment by yanivagman** [comment]
Don't forget to squash!

**Pull request merged**